firebase-database/src/main/java/com/google/firebase/database/connection/PersistentConnectionImpl.java

```diff
@@ -1049,6 +1066,32 @@ public void onResponse(Map<String, Object> response) {
         });
   }
 
+  private Task<Object> sendGet(final QuerySpec query, TaskCompletionSource<Object> source) {
+    Map<String, Object> request = new HashMap<String, Object>();
+    request.put(REQUEST_PATH, ConnectionUtils.pathToString(query.path));
+    request.put(REQUEST_QUERIES, query.queryParams);
+    sendAction(
+        REQUEST_ACTION_QUERY,
+        request,
+        new ConnectionRequestCallback() {
+          @Override
+          public void onResponse(Map<String, Object> response) {
+            String status = (String) response.get(REQUEST_STATUS);
+            if (status.equals("ok")) {
+              String pathString = (String) response.get(SERVER_DATA_UPDATE_PATH);
+              List<String> path = ConnectionUtils.stringToPath(pathString);
+              Object body = response.get(SERVER_DATA_UPDATE_BODY);
+              Long tagNumber = ConnectionUtils.longFromObject(response.get(SERVER_DATA_TAG));
```

Comment:
Does this need to be added to the Web PR?

Reply:
No I think this was a mistake in this PR. Gets won't send tags.